src/kv/LevelDBStore.cc

```diff
@@ -187,6 +196,24 @@ void LevelDBStore::close()
     cct->get_perfcounters_collection()->remove(logger);
 }
 
+int LevelDBStore::repair(std::ostream &out)
+{
+  leveldb::Options ldoptions;
+  int r = load_leveldb_options(false, ldoptions);
+  if (r) {
+    dout(1) << "load leveldb options failed" << dendl;
+    out << "load leveldb options failed" << std::endl;
+    return r;
+  }
+  leveldb::Status status = leveldb::RepairDB(path, ldoptions);
+  if (status.ok()) {
+      return 0;
```

Comment:
wrong indent.

Reply:
sorry, I will ping you after I fix all those problems. I will review my patch carefully.

Reply:
it seems that I have fixed all indent problems. there are some blank lines(I left one blank line between two functions), need I remove those blank lines?